.env.example

```
@@ -54,3 +54,7 @@ ADMIN_AUTH_SECRET_KEY=thisisadminsecret
 ADMIN_AUTH_TOKEN_DURATION=1h
 ADMIN_AUTH_ISSUER=cashdash
 ADMIN_AUTH_HASH_COST=10
+
+PAYMENT_GATEWAY=midtrans
+PAYMENT_SERVER_KEY=
+PAYMENT_ENVIRONMENT=sandbox
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PAYMENT_ENVIRONMENT key should go before the PAYMENT_GATEWAY key

(UnorderedKey)